src/Pages/HowToPlay.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import '../App.css';
+import '../Styles/HowToPlay.css';
+import Navbar from '../Components/Navbar/Navbar'
+
+function HowToPlay() {
+
+  React.useEffect(() => {
+    document.body.style.overflow = 'hidden';
```

Comment:
why wouldn't you define this in a css file

Reply:
this is how I get the css to apply only when this page is up, you cant do that in the css file. Makes it so that my code works, and yours isnt broken